File: mcached/java/update/mcached_update.java
// --------------------------------------------------------------
//	mcached_update.java
//
//					Feb/04/2015
// --------------------------------------------------------------
import	java.util.HashMap;

// --------------------------------------------------------------
public class mcached_update
{
 
// --------------------------------------------------------------
public static void main (String[] args) throws Exception
{
	System.out.println ("*** 開始 ***");

	String	key = args[0];
	int	population = Integer.parseInt (args[1]);
	System.out.print ("\tkey = " + key);
	System.out.println ("\tpopulation = " + population);

	final String server = "localhost";
	final int port = 11211;

	mcached_socket.mcached_update_proc (server,port,key,population);

	System.out.println ("*** 終了 ***");
}

// --------------------------------------------------------------
}
// --------------------------------------------------------------

File: cdb/rhino/update/mcached_update.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// --------------------------------------------------------------
//	mcached_update.js
//
//					Feb/05/2015
// --------------------------------------------------------------
importPackage (java.util)
importPackage (java.io)
importPackage (java.net)

load ("/var/www/data_base/common/rhino_common/mcached_manipulate.js") 
load ("/var/www/data_base/common/rhino_common/text_manipulate.js") 
// --------------------------------------------------------------
print ("*** 開始 ***")

var key_in=arguments[0]
var population_in=arguments[1]
print	("key_in = " + key_in + "\tpopulation_in = " + population_in)

var server = "localhost"
var port = 11211

mcached_update_proc (server,port,key_in,population_in)

print ("*** 終了 ***")
 
// --------------------------------------------------------------
